docker/Dockerfile.webrequest

```diff
@@ -1,26 +1,36 @@
 FROM --platform=linux/amd64 python:3.12.6-slim
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends wget gnupg && \
-    wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - && \
-    echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list && \
+ENV base=/app
+ENV path=webrequest
+ENV selpath=selenium
+
+RUN mkdir -p "${base}/${path}" && \
+    mkdir -p "${base}/${selpath}" && \
     apt-get update && \
-    apt-get install -y --no-install-recommends google-chrome-stable && \
+    apt-get install -y --no-install-recommends wget gnupg ca-certificates unzip \
+        openjdk-17-jre-headless fonts-liberation libasound2 libatk-bridge2.0-0 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libglib2.0-0 libgtk-3-0 libnspr4 libnss3 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 lsb-release xdg-utils && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
-    useradd -m appuser
+    useradd -m appuser && \
+    mkdir -p /selenium && \
+    cd "${base}/${selpath}" && \
+    wget -q https://storage.googleapis.com/chrome-for-testing-public/128.0.6613.119/linux64/chrome-linux64.zip && \
+    wget -q https://storage.googleapis.com/chrome-for-testing-public/128.0.6613.119/linux64/chromedriver-linux64.zip && \
+    wget -q https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.24.0/selenium-server-4.24.0.jar && \
+    unzip chromedriver-linux64.zip && \
+    unzip chrome-linux64.zip && \
+    rm chromedriver-linux64.zip chrome-linux64.zip
 
```

Comment:
could you add some comments explaining why we need this?

Reply:
added in b25162c06821ecec8c034d5e465d4ab15e719d6b